Kimseho_assignment/app.js

```diff
@@ -0,0 +1,73 @@
+require("dotenv").config();
+
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
```

Comment:
- 추후 에러 핸들링을 담을 때의 가독성 및 확장성 보장을 위해 ".then"을 (typeorm datasource 공식문서에서 제안하는 것 처럼) 한칸 아래로 띄는 것을 권장드립니다.
 -
```suggestion
myDataSource.initialize()
        .then(() => {
         console.log("Data Source has been initialized!");
});
```